steps/apps-decompiler/1.0.0/step.yml

```diff
@@ -0,0 +1,79 @@
+title: Mobile apps decompiler
+summary: |
+  Decompile mobile APK and/or IPA to look inside
+description: |
+  Decompile mobile APK and/or IPA to look inside
+
+  ### Troubleshooting
+
+  - Be aware, you have to launch this step after the compilation step ("Android Build" for android, "Xcode Archive" for iOS)
+  - Launch "Deploy to Bitrise.io" step after this one to deploy report files to artifacts
+website: https://github.com/odemolliens/bitrise-step-apps-decompiler
+source_code_url: https://github.com/odemolliens/bitrise-step-apps-decompiler
+support_url: https://github.com/odemolliens/bitrise-step-apps-decompiler/issues
+published_at: 2021-01-21T12:04:43.593758+01:00
+source:
+  git: https://github.com/odemolliens/bitrise-step-apps-decompiler.git
+  commit: 9d55d0cb5e332c4be62952c78329efbf8dd5f8e8
+host_os_tags:
+- osx-10.10
+- ubuntu-16.04
+project_type_tags:
+- ios
+- android
+- react-native
+- flutter
+type_tags:
+- utility
+toolkit:
+  bash:
+    entry_file: step.sh
+deps:
+  brew:
+  - name: jq
+  - name: apktool
+is_requires_admin_user: false
+is_always_run: true
+is_skippable: false
+inputs:
+- decompile_android: "yes"
+  opts:
+    is_expand: true
+    is_required: true
+    summary: Do you want decompile Android APK?
+    title: Decompile Android APK
+    value_options:
+    - "yes"
+    - "no"
+- decompile_ios: "yes"
+  opts:
+    is_expand: true
+    is_required: true
+    summary: Do you want decompile iOS IPA?
+    title: Decompile iOS IPA
+    value_options:
+    - "yes"
+    - "no"
+- opts:
+    description: |
+      You can use the artifacts from another bitrise build of the same project.
+      If you launch this step on the build where you launched Android build step eg. you don't have to set this key
+    is_expand: true
+    is_required: false
+    summary: Build slug of the build where you have the APK/IPA into artifacts
+    title: External Build slug
+  outside_build_slug: null
+- android_apk_path: $BITRISE_APK_PATH
+  opts:
+    description: |
+      If you use this step in the same build where you generated the APK
+    is_expand: true
+    is_required: true
+    title: APK path
+- ios_ipa_path: $BITRISE_IPA_PATH
+  opts:
+    description: |
+      If you use this step in the same build where you generated the IPA
```

Comment:
The wording sounds strange. Is a calque?
It's jus an IPA path? What is the point of "If"?

Reply:
The user will be able to launch this step from the step where he will generate his IPA
In that case, we need to have the path to get it. Per default `XCode archive` step save the path in `$BITRISE_IPA_PATH` but to be able to support the case where the user deployed it in another path we have this input.

In the case where the user launches this step outside of the step where he will generate the IPA (eg. to launch a build specifically for quality to not mix it with deploy build) also we don't need to have this path because it will download the IPA directly from the deployment builds artifacts with `outside_build_slug` input

Reply:
OK, so you should somehow express it in a condensed way for a valid description.